sky/global_user_state.py

```diff
@@ -1617,7 +1617,6 @@ def get_clusters(
                 cluster_table.c.status, cluster_table.c.autostop,
                 cluster_table.c.to_down, cluster_table.c.owner,
                 cluster_table.c.metadata, cluster_table.c.cluster_hash,
-                cluster_table.c.storage_mounts_metadata,
```

Comment:
Add a comment to the else block to mention that `storage_mounts_metadata` is not included above? Just like the line with `# extra fields compared to above query`